src/components/ArticleCard.jsx

```diff
@@ -1,10 +1,17 @@
-const ArticleCard = ({title, article_img_url, votes}) => {
-    
-    return <section className="article-thumbnails">
-        <strong>{title}</strong><br /><br />
+import { Link } from "react-router-dom";
+
+const ArticleCard = ({ title, article_img_url, votes, article_id }) => {
+
+  return (<Link to={`articles/${article_id}`}>
+      <div className="article-thumbnails">
+        <strong>{title}</strong>
+        <br />
```

Comment:
this isn't best practice to create spaces between lines, you should be able to give different tags classes and change the margin/padding around them to make a break